avro/src/test/scala/magnolify/avro/test/AvroTypeSuite.scala

```diff
@@ -98,42 +97,37 @@ class AvroTypeSuite extends MagnolifySuite {
     }
   }
 
+  import magnolify.scalacheck.test.TestArbitrary._
```

Comment:
jw, why not put these in the top-level import block?

Reply:
I wanted to group the implicts for/under test here but can be moved in the top level